src/server/webhookServer.ts

```diff
@@ -0,0 +1,74 @@
+import * as http from 'http';
+import _debug from 'debug';
+import { LIBRARY_NAME } from '../constants';
+import { IAlice } from '../alice';
+
+const debug = _debug(LIBRARY_NAME);
+
+export interface IWebhookServerConfig {
+  port: number;
+  webhookUrl: string;
+  options: object;
+  handleRequest: IAlice['handleRequest'];
+}
+export interface IWebhookServer {
+  start(): void;
+  stop(): void;
+}
+
+export class WebhookServer {
+  private server: http.Server;
+  private port: number;
```

Comment:
Please add underscore before private fields. As soon as this lib will be used not only in type script i think it's good idea to have private marker not only as field modificator, but as name prefix.

Reply:
done. Actually I'm not true fan of underscores in private methods, bc this `webhookServer` won't be used outside the library, but anyways, okay, maybe maybe